app/lib/release_phase_notifier.rb

```diff
@@ -0,0 +1,11 @@
+class ReleasePhaseNotifier
+  def self.ping_slack
+    client = Slack::Notifier.new(
+      ApplicationConfig["SLACK_WEBHOOK_URL"],
+      channel: ApplicationConfig["SLACK_DEPLOY_CHANNEL"],
+      username: "Heroku",
+    )
```

Comment:
I get `undefined method `request_uri' for #<URI::Generic >` when I run this, because by default `SLACK_WEBHOOK_URL` is empty, I think we should bail from this method if they aren't both defined

Reply:
Took it a step further and added error handling for Slack API errors as well so if Slack goes down we can still deploy. I would be persuaded to catch any `StandardError` if you all think that would be better.